pkg/image/registry/imagestream/strategy.go

```diff
@@ -462,21 +463,24 @@ func (v *TagVerifier) Verify(old, stream *imageapi.ImageStream, user user.Info)
 		}
 
 		// Make sure this user can pull the specified image before allowing them to tag it into another imagestream
-		subjectAccessReview := authorizationapi.AddUserToSAR(user, &authorizationapi.SubjectAccessReview{
-			Action: authorizationapi.Action{
-				Verb:         "get",
-				Group:        imageapi.LegacyGroupName,
-				Resource:     "imagestreams/layers",
-				ResourceName: streamName,
+		subjectAccessReview := authorizationutil.AddUserToSAR(user, &authorizationapi.SubjectAccessReview{
+			Spec: authorizationapi.SubjectAccessReviewSpec{
+				ResourceAttributes: &authorizationapi.ResourceAttributes{
+					Namespace:   tagRef.From.Namespace,
+					Verb:        "get",
+					Group:       imageapi.GroupName,
```

Comment:
I change this.  Looks like a typo before.  This groupified check should also have permission.

Reply:
does empty group vs. image.openshift.io matters here? guess we need support both

Reply:
> does empty group vs. image.openshift.io matters here? guess we need support both

It's a server back to itself call, the roles will be reconciled.  I think this usage is correct.

Reply:
thx for clarification :)